container/image.bzl

```diff
@@ -352,23 +351,16 @@ def _repository_name(ctx):
     return _join_path(ctx.attr.repository, ctx.label.package)
 
 def _assemble_image_digest(ctx, name, image, image_tarball, output_digest):
-    args = ctx.actions.args()
-    blobsums = image.get("blobsum", [])
-    args.add_all(blobsums, format_each = "--digest=%s")
-    blobs = image.get("zipped_layer", [])
-    args.add_all(blobs, format_each = "--layer=%s")
-    args.add(image["config"], format = "--config=%s")
-    args.add(output_digest, format = "--output-digest=%s")
-
-    if image.get("legacy"):
-        args.add(image["legacy"].path, format = "--tarball=%s")
+    args, inputs = _gen_img_args(ctx, image)
+    args.append("--dst=%s" % output_digest.path)
```

Comment:
create an args object and add_all the args produced by _gen_img_args but then you dont use append to add the output_digest, which should be slightly better wrt efficiency

Reply:
Done